kqueen/storages/etcd.py

```diff
@@ -132,11 +132,16 @@ def _unpad(self, s):
     def encrypt(self):
         """Encrypt stored value"""
 
+        serialized = self.serialize()
+
         if not self.encrypted:
-            return self.serialize()
+            return serialized
+
+        if self.value is None:
```

Comment:
Isn't it possible to have required=False attribute on model with encrypted=True? This feels like it's going to raise unnecessarily

Reply:
Good point ... `serialize` returns `None` for missing value and `encrypt` should do it same way.